packages/suite/src/hooks/suite/__tests__/useCountdownTimer.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { act, render } from '@testing-library/react';
+import { fixtures } from 'src/hooks/suite/__fixtures__/useCountdownTimer';
+import { useCountdownTimer } from 'src/hooks/suite/useCountdownTimer';
+
+type Result = ReturnType<typeof useCountdownTimer>;
+
+const Component = ({
+    params,
+    callback,
+}: {
+    params: Parameters<typeof useCountdownTimer>;
+    callback: (res: Result) => void;
+}) => {
+    const result = useCountdownTimer(...params);
+    callback(result);
+    return null;
+};
+
+describe('useCountdownTimer', () => {
```

Comment:
Thanks for the test. Did you add it for coverage? Please put it in separate commit, it's not related at all to RBF in exchange 🙏

Reply:
Yes, fixed in https://github.com/trezor/trezor-suite/pull/9939/commits/3335536c4c736c4d909d0772f96d7a56c44399a7